collector/logical_port_collector.go

```diff
@@ -15,18 +18,26 @@ type logicalPortCollector struct {
 	client *nsxt.APIClient
 	logger log.Logger
 
-	logicalPortTotal *prometheus.Desc
-	logicalPortUp    *prometheus.Desc
+	logicalPortTotal  *prometheus.Desc
+	logicalPortUp     *prometheus.Desc
+	logicalPortStatus *prometheus.Desc
 }
 
 func newLogicalPortCollector(client *nsxt.APIClient, logger log.Logger) prometheus.Collector {
 	logicalPortTotal := prometheus.NewDesc(prometheus.BuildFQName(namespace, "logical_port", "total"), "Total number of logical ports.", nil, nil)
 	logicalPortUp := prometheus.NewDesc(prometheus.BuildFQName(namespace, "logical_port", "up"), "Number of logical ports currently up.", nil, nil)
+	logicalPortStatus := prometheus.NewDesc(
```

Comment:
nit: lets make it consistent with `logicalPortTotal` and `logicalPortUp` (multilines or single line)

Reply:
ack, will make it multi-lines as desc initialization will grow with the number of labels